src/notifications/mapTransactionsType.ts

```diff
@@ -1,30 +1,10 @@
-import { NotificationTransactionTypes, NotificationTransactionTypesType } from '@/notifications/types';
-import { TransactionType } from '@/entities';
+import { NotificationTransactionTypesType } from '@/notifications/types';
+import { TransactionType } from '@/resources/transactions/types';
 
 /**
  * Mapping of notification backend transaction types sent in notification
  * payload, to internal `TransactionType` that we use in the app already.
  */
 export function mapNotificationTransactionType(notificationTransactionType: NotificationTransactionTypesType): TransactionType {
```

Comment:
dont really need these mappings anymore since both have the same data source and match up